force-app/main/default/objects/Case/fields/EDRD_Funding_Duration_Unit__c.field-meta.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>EDRD_Funding_Duration_Unit__c</fullName>
+    <externalId>false</externalId>
+    <label>Funding Duration Unit</label>
+    <required>false</required>
+    <trackFeedHistory>false</trackFeedHistory>
+    <trackHistory>false</trackHistory>
+    <trackTrending>false</trackTrending>
+    <type>Picklist</type>
+    <valueSet>
+        <restricted>true</restricted>
+        <valueSetDefinition>
+            <sorted>false</sorted>
+            <value>
+                <fullName>once</fullName>
+                <default>false</default>
```

Comment:
The picklist values need to be updated.  Once, Days, Weeks, Months, Years @rahulshyamrajsingh @deepakmulamalla

Reply:
The field's label has been changed because the build is giving us issues when we attempt to modify the API name.